index.html

```diff
@@ -176,13 +176,13 @@ <h1 class="light section-title">Why Spine</h1>
                     <i class="fas fa-lightbulb-on card-icon-font"></i>
                     <div class="card-content-text">
                         <header class="feature-card-title">Clear API</header>
-                        <p>Concepts known from the DDD books right are right in the code.
-                            <code><a href="https://spine.io/core-java/javadoc/server/io/spine/server/aggregate/Aggregate.html">Aggregate</a></code>,
+                        <p>Concepts from the DDD books, such as
+                            <code><a href="hOpen for Business Changesttps://spine.io/core-java/javadoc/server/io/spine/server/aggregate/Aggregate.html">Aggregate</a></code>,
```

Comment:
I think the text after `h` was intended to somewhere else.

Reply:
LGTM.